lib/apipie/swagger_generator.rb

```diff
@@ -269,7 +271,9 @@ def remove_colons(str)
     end
 
     def swagger_op_id_for_method(method)
-      remove_colons method.resource.controller.name + '::' + method.method
+      op_id = remove_colons method.resource.controller.name + '::' + method.method
```

Comment:
nit: Seems like we'd have more flexibility if this part was also done in the OpIdFormatter and it just generated the whole op_id from the method/path.

Reply:
I originally had this in there, and then pulled it out as I think it's a more flexible approach.  1 - it follows the same convention as the RoutesFormatter, where the route is already formatted a standard way, and then you can add a handler to provide additional formatting.  2 - this is a little more extensible.  This approach reduces the amount of changes to the core apipie functionality.  If we want to upgrade apipie versions, having this logic not exist in the formatter will make that easier.

Reply:
Scratch point 1, but point 2 still stands